newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -977,6 +991,34 @@ export const LeaderboardAdmin = ({ onLoading, project }: Props) => {
               }}
             />
           ) : null}
+          {isEditingSortOptions ? (
+            <LeaderboardSortOptionsDialog
+              open
+              onClose={() => setIsEditingSortOptions(false)}
+              onSave={async (sortOptions: {|
+                sort: LeaderboardSortOption,
+                extremeAllowedScore: ?number,
+              |}) => {
+                try {
+                  await onUpdateLeaderboard(i18n, {
+                    ...sortOptions,
+                  });
+                } finally {
+                  setIsEditingSortOptions(false);
+                }
+              }}
+              sort={
+                isEditingSortOptions && currentLeaderboard
```

Comment:
Sorry I was not clear 😅 
I meant that the dialog should not be open if there is no current leaderboard. So you could have:

```js
{isEditingSortOptions && currentLeaderboard ? (
            <LeaderboardSortOptionsDialog ... />) : null}
```
This way, you can avoid the checks on `currentLeaderboard` definition in the props, and giving the default `ASC` value that is not "true"

Reply:
oh yes alright !